datalab/datalab_session/analysis/line_profile.py

```diff
@@ -9,9 +10,32 @@ def line_profile(input: dict):
     Creates an array of luminosity values and the length of the line in arcseconds
   """
   sci_hdu = get_hdu(input['basename'], 'SCI')
-
+  
   x_points, y_points = scale_points(input["height"], input["width"], sci_hdu.data.shape[0], sci_hdu.data.shape[1], x_points=[input["x1"], input["x2"]], y_points=[input["y1"], input["y2"]])
+  
+  # Line profile and distance in arcseconds
   line_profile = profile_line(sci_hdu.data, (x_points[0], y_points[0]), (x_points[1], y_points[1]), mode="constant", cval=-1)
-  arcsec = len(line_profile) * sci_hdu.header["PIXSCALE"]
+  
+
+  # Calculations for coordinates and angular distance
+  try: 
+    wcs = WCS(sci_hdu.header)
+
+    start_sky_coord = wcs.pixel_to_world(x_points[0], y_points[0])
+    end_sky_coord = wcs.pixel_to_world(x_points[1], y_points[1])
+
+    arcsec_angle = start_sky_coord.separation(end_sky_coord).arcsecond
+
+    start_coords = [start_sky_coord.ra.deg, start_sky_coord.dec.deg]
+    end_coords = [end_sky_coord.ra.deg, end_sky_coord.dec.deg]
+
+  except Exception as e:
+    print(e)
+    error = f'{input["basename"]} does not have a valid WCS header'
```

Comment:
This error string isn't being logged anywhere it seems? Not necessary for this PR, but we should probably be using python/django logging: https://medium.com/@torkashvand/a-comprehensive-guide-to-logging-in-django-e041f311bcb7

Reply:
I left this here as a place holder for the error reporting story in the future, but I can print it out in the meantime